2018-09-10-was-ist-das-xcamp-und-wer-sollte-teilnehmen.md

```diff
@@ -0,0 +1,18 @@
+---
+title:     Video-Interview: Was ist das XCamp und wer sollte teilnehmen?
+author:    Jens Bothmer
+published: 10.09.2019
+image:     
+tags:      XCamp2019, Sessions
+---
+
+Interview mit Dr. Christiane Gerigk und Jens Bothmer – Initiatoren des Xcamps.
+
+Termin 13.09.2019
+Frankfurt am Main
+Skydeck Silberturm, 31. Etage
+bei DB Systel GmbH
+
+<a class="ytp-title-link yt-uix-sessionlink" target="_blank" data-sessionlink="feature=player-title" href="https://www.youtube.com/watch?v=4HbXMuR2eN0">Was ist das XCamp? Interview Dr. Christiane Gerigk und Jens Bothmer</a>
```

Comment:
Couldn't we just use an `<iframe>` here? Youtube tells us how:
```
<iframe width="560" height="315" src="https://www.youtube.com/embed/4HbXMuR2eN0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Reply:
Yes of course! iFrame added :) I added as well the Text "Was ist das XCamp? Interview Dr. Christiane Gerigk und Jens Bothmer", but I don't know if it's ok... Does You Tube add it by it self?

Reply:
Actually, I don't know, let's just try. 😁